source/components/the-component-lifecycle.md

```diff
@@ -1,25 +1,137 @@
-Part of what makes components so useful is that they let you take
-complete control of a section of the DOM.  This allows for direct DOM
-manipulation, listening and responding to browser events, and using 3rd
-party JavaScript libraries in your Ember app.
+Part of what makes components so useful is that they let you take complete control of a section of the DOM.
+This allows for direct DOM manipulation, listening and responding to browser events, and using 3rd party JavaScript libraries in your Ember app.
 
-As components are rendered, re-rendered and finally removed, Ember
-provides _lifecycle hooks_ that allow you to run code at specific
-times in a component's life.
+As components are rendered, re-rendered and finally removed, Ember provides _lifecycle hooks_ that allow you to run code at specific times in a component's life.
 
-To get the most use out of a component, it is important to understand
-these lifecycle methods. The following hooks are a few of the most
-useful and commonly used in Ember apps.
+To get the most use out of a component, it is important to understand these lifecycle methods.
 
-## Attaching to the Component Element
+To make use of these hooks simply override them within your component.
```

Comment:
I'm not too keen on this bit, nor the note about `_super`. I think https://guides.emberjs.com/v2.2.0/object-model/classes-and-instances/#toc_overriding-parent-class-methods should cover it sufficiently.

Maybe replace the example with something like "We will know see which hooks exist, what order and when are they fired and then in what situations you'd want to use them"?